snakemake/maize_allele_frequency_eval/dataset/README.md

```diff
@@ -1,11 +1,12 @@
 # Maize allele frequency eval dataset creation
```

Comment:
Should this pipeline live in `/experiments`?  I see https://github.com/plantcad/plantcad-dev/pull/27#issuecomment-3390819250, and it makes sense to me that we want this to be something that isn't experimental (i.e. be in `/src`).  Given https://github.com/plantcad/plantcad-dev/pull/26 though, I don't see why this shouldn't be implemented that way with the main advantages being config validation, multi-node support, linting, it's more testable (i.e. not sure how to do that with inline python in snakemake rules), Ray would be available, etc.

What do you think of keeping this in experiments and then coming up with some plan to make it consistent with the other evals?  I'd be happy to work on that or pair with you for it.